static/js/components/tiles/sv_ranking_units.tsx

```diff
@@ -109,6 +108,7 @@ export function SvRankingUnits(props: SvRankingUnitsProps): JSX.Element {
             rankingGroup,
             rankingMetadata,
             true,
+            props.apiRoot,
```

Comment:
@dwnoble Can you help check if all the use of "apiRoot" here and below makes sense?